### ISSUE TITLE
Schema diagram

### ISSUE THREAD

**Issue by collinschwantes** [ecohealthalliance/airtabler#34]
**Describe why this feature should be added**
It is often helpful to visualize schemas a graphs to see relationships between entities 

**Describe the behavior you expect**
Given a schema object, create a graph showing the relationships between entities. Edges == linkedRecords, Nodes == Tables. Node labels == fields

**Additional context**
the DM package may be helpful to guide this process.

**Comment by collinschwantes** [comment]
@ernestguevarra Does this look sufficient?

**Comment by ernestguevarra** [comment]
Yes, for what I think will be useful for what I am doing now, this is more than enough!
